### ISSUE TITLE
crypto/tls: bogo suite on Windows is flakey

### ISSUE THREAD

**Issue by rolandshoemaker** [golang/go#66913]
See e.g. https://ci.chromium.org/ui/p/golang/builders/ci/gotip-windows-amd64-longtest/b8750306255775534897/test-results?sortby=&groupby=

Seems like another case of Windows just not liking creating a lot of connections, unclear what a real solution here is. Probably just disabling it on windows builders?

**Comment by matloob** [comment]
I think it would be a good idea to skip it on Windows for now.

**Comment by rolandshoemaker** [comment]
Yup, will send a CL. I have a possible plan to make bogo skip the network stack entirely and use in-memory connections, but that will require a bunch of work in boringssl.
